### DISCUSSION TITLE
Building Companion on MacOS - what am I missing?

### DISCUSSION THREAD

**Discussion by philmoz** [EdgeTX/edgetx#2820] [Q&A]
I've set up the build on MacOS 12.6.1 as per the Wiki and 'make all' runs without any errors.

However the resulting companion.app bundle only contains the MacOS/companion and info.plist files. It is missing the Frameworks, Plugins and Resources folders.

Any advice on what I have missed would be appreciated.

**Comment by philmoz** [comment]
Never mind, I worked it out.

**Comment by raphaelcoeffic** [comment]
@philmoz is there anything we should add to the Wiki related to that?

**Comment by philmoz** [comment]
I am able to rebuild the app; but I had to mess with it a lot to get it to work. I’m still trying to work out the build intricacies to the see if I can to it more simply. There are issues with SDL versioning and the way dfu-util is installed by homebrew. If/when I get it working better I will post details.

**Reply by raphaelcoeffic** [reply]
I made some attempt at using SDL2 instead of the old, and it seemed fairly easy. I should have the branch somewhere lying around...
